app/src/main/java/com/github/se/wanderpals/ui/screens/trip/Map.kt

```diff
@@ -85,15 +91,22 @@ fun Map(tripId: String, mapViewModel: MapViewModel) {
       expanded = false
       return@LaunchedEffect
     }
-    requestLocation(
-        searchText,
-        onSuccessRequest = { output, loc ->
-          // take all the suggestions
-          listOfPropositions = output
+    getAddressPredictions(
+        client,
+        inputString = searchText,
+        location = LatLng(46.519653, 6.632273),
```

Comment:
It should be a variable, even though it would be later replaced by the position of the city where the trip is originally planned or the coordinate of the map view

Reply:
Will try to replace it with the camera's coordinates